### ISSUE TITLE
Script to add new RF4 projects/artifacts to OSO project registry

### ISSUE THREAD

**Issue by ccerv1** [opensource-observer/oso#1207]
### What is it?

We need to by syncing with the Agora RF4 sign-up database and handling the following cases:

Case 0: project is already on OSO --> link Agora UID with project ID
Case 1: new valid project; none of the artifacts have been claimed --> add project and artifacts to OSO
Case 2: existing project, new artifact --> add artifact to project on OSO
Case 3: sub-team from within a larger project --> prompt decision: either ignore (treat as Case 0) or split the parent project

**Comment by ryscheng** [comment]
I think the current plan is to just get a CSV of registrations from Agora, convert it to yaml, merge into oss-directory.

Once indexing is done, Agora will query our API and snapshot it into their own database for voting

**Comment by ccerv1** [comment]
As discussed earlier @ryscheng - here is a [link](https://drive.google.com/file/d/16sSCX-1Qi2mexUseGZkVjWn7SFmfm966/view?usp=drive_link) to the CSV file. You'll see there is essentially one row per artifact and columns for `URL` (GitHub and NPM links I believe) and also for `deployer` and `contract`

**Comment by ryscheng** [comment]
Initial transformation here for existing projects
https://github.com/opensource-observer/oss-directory/pull/367

Subsequent PR with the actual run.

**Comment by ryscheng** [comment]
Minor fix around trim
https://github.com/opensource-observer/oss-directory/pull/368

**Comment by ryscheng** [comment]
write new files and merge projects with de-dupe
https://github.com/opensource-observer/oss-directory/pull/377/files

**Comment by ryscheng** [comment]
Add JSON support here
https://github.com/opensource-observer/oss-directory/pull/378

**Comment by ryscheng** [comment]
Closing for now I thnk the script is where we need it
